run_local_with_dependencies.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-sm2 -start MONGO DATASTREAM EMAIL HMRC_EMAIL_RENDERER MAILGUN_STUB DESKPRO_STUB DESKPRO_TICKET_QUEUE
+sm2 -start MONGO DATASTREAM 
+
+# sm2 -start EMAIL HMRC_EMAIL_RENDERER MAILGUN_STUB DESKPRO_STUB DESKPRO_TICKET_QUEUE
```

Comment:
It might be helpful to change `MAILGUN_STUB` to `DIGITAL_CONTACT_STUB` during this edit.